modules/Microsoft.Web/staticSites/config/deploy.bicep

```
@@ -0,0 +1,50 @@
+@allowed([
+  'appsettings'
+  'functionappsettings'
+])
+@description('Required. Type of settings to apply.')
+param kind string
+
+@description('Required. App settings.')
+param properties object
+
+@description('Conditional. The name of the parent Static Web App. Required if the template is used in a standalone deployment.')
+param name string
```

For consistency with other children 
```suggestion
param staticSiteName string
```